mlrun/alerts/alert.py

```diff
@@ -74,11 +74,17 @@ def validate_required_fields(self):
     def to_dict(self, fields: list = None, exclude: list = None, strip: bool = False):
         data = super().to_dict(self._dict_fields)
 
+        if self.entities is None:
```

Comment:
I don't see an issue with the error but the way you use `to_dict` to parse the entities and notifications is not really the mlrun way. you should use `_serialize_field` see `ModelObj._fields_to_serialize`